src/api/status/src/server.js

```diff
@@ -1,15 +1,19 @@
 const { Satellite } = require('@senecacdot/satellite');
 const { static } = require('express');
 const path = require('path');
+var express = require('express');
+var cors = require('cors');
+var app = express();
 
+app.use(cors({ origin: '*' }));
```

Comment:
This probably has no effects. The server is started by Satellite. Please revert the changes.